### ISSUE TITLE
Updates not working in combination with cordova-plugin-crypt-file

### ISSUE THREAD

**Issue by datenwort** [markmarijnissen/cordova-app-loader#117]
Dear all,

are there known restrictions, if the plugin is used with the cordova-plugin-crypt-file? In the workflow of the app the files are decrypted during app run, but the update failed. 

Has anyone the same experience or a workaround?

**Comment by StefansArya** [comment]
Yeah, I have same issue with you. I haven't test it but **cordova-plugin-crypt-file** generate random key on each build and cordova-app-loader doesn't encrypt/decrypt the downloaded file.

But I think you can upload the non-encrypted files to the server, but it will be a security risk if some hacker know this vulnerability. Because the hacker can easily inject your application with their script and have a  control to your user's phone..
